### PR TITLE
Proofreading of tracepath manual

### PR DESCRIPTION
**Pull request by phkam**
A few suggestions to make the manpage more easy to read.

### CODE REVIEW

File: doc/tracepath.xml [hunk 1]
```diff
@@ -56,21 +56,21 @@ xml:id="man.tracepath">
     <info>
       <title>DESCRIPTION</title>
     </info>
-    <para>It traces path to
+    <para>It traces the networkpath to
```
Comment (zenczykowski) [inline doc/tracepath.xml]: networkpath is not a word - needs space

Comment (phkam) [reply doc/tracepath.xml]: Thank you for spotting this. I added a space and pushed again.

**Comment by pevik** [comment]
@phkam I'm going to merge it, could you please add your `Signed-off-by:` tag?
@zenczykowski I'll add your `Reviewed-by:`, ok? (with mail found on https://github.com/zenczykowski).

File: doc/tracepath.xml [hunk 2]
```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
  2:  3ffe:2400:0:109::2               463.514ms reached
      Resume: pmtu 1480 hops 2 back 2
     </literallayout>
-    <para>The first column shows TTL of the probe, followed by
-    colon. Usually value of TTL is obtained from reply from
-    network, but sometimes reply does not contain necessary
+    <para>The first column shows the TTL of the probe, followed by
+    colon. Usually the value of TTL is obtained from the reply from
+    the network. But sometimes it does not contain the necessary
```
Comment (pevik) [inline doc/tracepath.xml]: I'd keep here the original version: `from network, but sometimes` (compound sentence isn't forbidden in English :)).

File: doc/tracepath.xml [hunk 3]
```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
...
     <para>The second column shows the network hop, which replied to
-    the probe. It is either address of router or word [LOCALHOST],
-    if the probe was not sent to the network.</para>
+    the probe. It is either the address of the router or the word
+    [LOCALHOST], if the probe was not sent to the network.</para>
     <para>The rest of line shows miscellaneous information about
-    path to the corresponding network hop. As rule it contains
-    value of RTT. Additionally, it can show Path MTU, when it
+    the path to the corresponding network hop. At least it contains
+    the value of RTT. Additionally, it can show Path MTU, when it
```
Comment (pevik) [inline doc/tracepath.xml]: I'm not sure by some of the definite articles (`the`), but it's probably correct. A review from native speaker would be great.

File: doc/tracepath.xml [hunk 4]
```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
...
     changes. If the path is asymmetric or the probe finishes before
-    it reach prescribed hop, difference between number of hops in
-    forward and backward direction is shown following keyword
-    async. This information is not reliable. F.e. the third line
-    shows asymmetry of 1, it is because the first probe with TTL of
-    2 was rejected at the first hop due to Path MTU
+    it reaches the prescribed hop, the difference between number of
+    hops in forward and backward direction is shown following the
+    keyword "async". This information is not reliable. e.g. the
```
Comment (pevik) [inline doc/tracepath.xml]: `This information is not reliable, e.g.` (comma instead of dot).

Comment (pevik) [reply doc/tracepath.xml]: `the difference between number of hops in forward and backward direction is shown following the keyword "async". `
Not sure if it should be `shown with following the keyword "async".` or `shown by the keyword "async".`, but maybe I don't understand the meaning.

**Comment by phkam** [comment]
Thanks @pevik for your suggestions. I shall commit another revision beginning next week.

File: doc/tracepath.xml (hunk 2 above)
Comment (phkam) [reply doc/tracepath.xml]: Agreed. I reverted this part.

File: doc/tracepath.xml (hunk 4 above)
Comment (phkam) [reply doc/tracepath.xml]: According to my understanding it means `there is a number following the word "async"`.

**Comment by pevik** [comment]
Thanks, merged as 90cb393.

**Comment by pevik** [comment]
Sorry I forget to click to close PR.

**Pull request closed without merging**